terraform/ec2/variables.tf

```diff
@@ -21,7 +21,7 @@ variable "package_s3_bucket" {
 }
 
 variable "testing_ami" {
-  default = "amazonlinux2"
+  default = "amazonlinux3"
 }
```

Comment:
What is this default changing? For the sample app or the collector? Either way, I don't think we should change the default.

Reply:
The collector. But yeah, I agree. I changed default back to AL2022